README

```diff
@@ -11,6 +11,9 @@ infrastructure.
 [[building]]
 == Building
 
+You may get the source code from the
+https://github.com/Yubico/pam-u2f/tags[Release Tags].
```

Comment:
The signed tarballs (which include the pre-built manual pages) are only available through Yubico's website. What do you think of the following suggestion?

```suggestion
Signed release tarballs are available at Yubico's
https://developers.yubico.com/pam-u2f/Releases['release page'].
```

Reply:
ok